op-chain-ops/upgrades/l1.go

```diff
@@ -51,6 +54,27 @@ func L1CrossDomainMessenger(batch *safe.Batch, implementations superchain.Implem
 		return err
 	}
 
+	if chainConfig.ChainID == 10 || chainConfig.ChainID == 420 {
+		storageSetterABI, err := bindings.StorageSetterMetaData.GetAbi()
+		if err != nil {
+			return err
+		}
+		// https://github.com/ethereum-optimism/optimism/blob/86a96023ffd04d119296dff095d02fff79fa15de/packages/contracts-bedrock/.storage-layout#L28
+		calldata, err := storageSetterABI.Pack("setBytes32", common.Hash{31: 0xf9}, common.Hash{})
```

Comment:
why `0xf9` here?

Also, since we're using the storage setter here, should we take this opportunity to get rid of `CrossDomainMessengerLegacySpacer0`?

https://github.com/ethereum-optimism/optimism/blob/chain-ops/op-upgrade-2-step-mcp/packages/contracts-bedrock/src/universal/CrossDomainMessenger.sol#L15-L19